tests/custom_bootstrap.php

```diff
@@ -35,3 +35,9 @@
     'command' => 'doctrine:schema:create',
 ]);
 $application->run($input, new NullOutput());
+
+$input = new ArrayInput([
```

Comment:
we should add this everywhere to improve coverage , otherwise some classes get executed only on the previous commands.

Reply:
Do you plan to do the PR or should I ?

Reply:
If you can do them it would be great